source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,104 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Bulk conversion creates a job that queues and executes each query 
+conversion from a single request. You can select and convert one 
+or more queries, stored procedures, triggers, and views in bulk 
+by enabling the multi-select toggle. 
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+- .. include:: /includes/fact-query-converter-disclaimer.rst
+
+- .. include:: /includes/fact-view-query-conversion-bulk-status.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      From the :guilabel:`Code Generation` tab, click the 
+      :guilabel:`Query Converter` pane.
+
+   .. step:: Enable multi-selection
+
+      Click the :guilabel:`Enable multi-selection` toggle to show the query 
+      selection pane.
+
+   .. step:: Select your conversions
+
+      - To include or exclude a query from your bulk conversion job,
+        click the :icon-fa5:`check-square` icon next to the field name.
+
+      - To include or exclude entire categories from your download,
+        click :icon-fa5:`check-square` next to the category name.
+
+      - To include or exclude queries by their current status, use 
+        the :ref:`Filter<filter-queries>` to select a status.
+   
+   .. step:: Convert queries
+
+      Click the :guilabel:`Convert` button. Selected queries are 
+      queued for conversion and execute in batches. Total progress
+      of the conversion displays at the bottom-left of the page.
+         
+      - Individual query conversion status are displayed next to 
+        the query name.
+         
+      - The converted MongoDB syntax code can be viewed in the 
+        :guilabel:`Converted MongoDB Query` pane. 
+
+      - .. include:: /includes/fact-query-converter-failure-errors.rst
+
+      .. tip::
+
+         To stop a bulk conversion, click the red :guilabel:`Terminate` 
+         button in the lower-left corner of the screen and click the 
+         red :guilabel:`Terminate` button on the pop-up modal. Once a 
+         bulk job cancels, the bulk conversion status updates to 
+         ``Bulk convert job terminated``.
+
+
+   .. step:: (Optional) Stop the bulk conversion
```

Comment:
You have the text for stopping jobs twice, once as a tip and once as a step. We don't need both - given that people will rarely do this I think the tip might be the better choice.

Reply:
Apologies! That step was slated for removal after the tip was drafted.